classes/UpgradeContainer.php

```diff
@@ -664,13 +670,15 @@ public function getTwig()
                 $loader->addPath(realpath(__DIR__ . '/..') . '/views/templates', 'ModuleAutoUpgrade');
                 $twig = new Environment($loader);
                 $twig->addExtension(new TransFilterExtension3($this->getTranslator()));
+                $twig->addExtension(new AssetFunctionExtension3($this));
```

Comment:
Having the container as a parameter triggers the autowiring on the storybook side. Because of its `string` parameters, it cannot complete and makes the startup fail.